app/build.gradle

```diff
@@ -139,6 +137,12 @@ kotlin {
     experimental { coroutines 'enable' }
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        kotlinOptions.allWarningsAsErrors = true
```

Comment:
nit: I'd also expect the duplicated `kotlinOptions` to be unnecessary

Reply:
Thanks, fixed it :)